examples/javascript/cloudflare-worker/.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
+root = true
+
+[*]
+indent_style = tab
+end_of_line = lf
+charset = utf-8
+trim_trailing_whitespace = true
+insert_final_newline = true
+
+[*.yml]
+indent_style = space
```

Comment:
Should this have been included?

Reply:
It's autogenerated from the cloudflare scaffold, I see no harm in keeping it, but if we don't want it for some reason, I can remove it.

Reply:
I think remove, no need for too many extra files.